request/graphql/schema/types/descriptions.go

```diff
@@ -0,0 +1,223 @@
+`
+	commitLinkCIDFieldDescription string = `
+The CID of this linked commit.
+`
+	commitFieldsEnumDescription string = `
+These are the set of fields supported for grouping by in a commits query.
+`
+	commitsQueryDescription string = `
+Returns a set of commits matching any provided criteria, if no arguments are
+ provided all commits in the system will be returned.
+`
+	latestCommitsQueryDescription string = `
+Returns a set of head commits matching any provided criteria, if no arguments are
+ provided all head commits in the system will be returned. If no 'field' argument
+ is provided only composite commits will be returned. This is equivalent to
+ a 'commits' query with Depth: 1, and a differing 'field' default value.
+`
+	CountFieldDescription string = `
+Returns the total number of items within the specified child sets. If multiple child
+ sets are specified, the combined total of all of them will be returned as a single value.
+`
+	SumFieldDescription string = `
+Returns the total sum of the specified field values within the specified child sets. If
+ multiple fields/sets are specified, the combined sum of all of them will be returned as
+ a single value.
+`
+	AverageFieldDescription string = `
+Returns the average of the specified field values within the specified child sets. If
+ multiple fields/sets are specified, the combined average of all items within each set
+ (true average, not an average of averages) will be returned as a single value.
+`
+	booleanOperatorBlockDescription string = `
+These are the set of filter operators available for use when filtering on Boolean
+ values.
+`
+	notNullBooleanOperatorBlockDescription string = `
+These are the set of filter operators available for use when filtering on Boolean!
+ values.
+`
+	dateTimeOperatorBlockDescription string = `
+These are the set of filter operators available for use when filtering on DateTime
+ values.
+`
+	floatOperatorBlockDescription string = `
+These are the set of filter operators available for use when filtering on Float
+ values.
+`
+	notNullFloatOperatorBlockDescription string = `
+These are the set of filter operators available for use when filtering on Float!
+ values.
+`
+	intOperatorBlockDescription string = `
+These are the set of filter operators available for use when filtering on Int
+ values.
+`
+	notNullIntOperatorBlockDescription string = `
+These are the set of filter operators available for use when filtering on Int!
+ values.
+`
+	stringOperatorBlockDescription string = `
+These are the set of filter operators available for use when filtering on String
+ values.
+`
+	notNullStringOperatorBlockDescription string = `
+These are the set of filter operators available for use when filtering on String!
+ values.
+`
+	idOperatorBlockDescription string = `
+These are the set of filter operators available for use when filtering on ID
+ values.
+`
+	eqOperatorDescription string = `
+The equality operator, if the target matches the value the check will pass.
```

Comment:
todo: From here to line 205, I would change `, if` to `. If`.

Reply:
It feels like it should be all part of the same sentence. "-" is totally fine too.

Reply:
changed